From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-cmans__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-cmans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

azurecla commented:
Hi __@v-cmans__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-cmans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
acrolinx

Commit:
links

Commit:
moving snapshots back into the files topic

Commit:
pre-review

Commit:
snapshots back into files topic

Commit:
moving snapshots into files topic

Commit:
fixing checklist

Commit:
fixing checklist

Commit:
fixing checklist

Commit:
acrolinx

Commit:
snapshots content

Commit:
removing unused art and incorping Tamras review feedback

Commit:
quickstart and some of Tamras review that applies to this file

Commit:
quickstart

Commit:
spaces

Commit:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

Commit:
testing

wmgries committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into scripting-quickstarts

wmgries committed:
Manually bringing in some changes over from @cynthn/feedback from @tamram.

wmgries committed:
Incorporate feedback from @tamram and @cynthn into the PowerShell guide.

wmgries committed:
Incorporate feedback from @tamram and @cynthn into the Portal guide.

wmgries committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into scripting-quickstarts

wmgries committed:
Fix broken links

wmgries committed:
"Azure File (s|S)hare(s|)" => "Azure file share".

Commit:
Tamra feedback

Commit:
mac os

Commit:
mac os

Commit:
safekeeping

Commit:
fixing heading

Commit:
drafting

Commit:
fixing art

Commit:
intro

Commit:
art

Commit:
fixing branding

Commit:
fixing branding

Commit:
branding

Commit:
branding

dhaval24 committed:
Update JRE version requirements, fix code snippet to manually pass instrumentation key

dhaval24 committed:
update agent documentation - modified title to be relevant, updated dependency support and added statement to indicate version matching between agent and core or web sdk

dhaval24 committed:
Update the app-insights-api-custom-event-metrics with java examples

dhaval24 committed:
Updating sampling docs to include fixed rate sampling in java sdk

dhaval24 committed:
Update trouble shooting guidelines for Java

dhaval24 committed:
Fix broken Links

Commit:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

Commit:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

dhaval24 committed:
addressing feedback

mimig1 committed:
Cosmos DB: TOC & index for public A/B experiment

mimig1 committed:
updates

mimig1 committed:
update

mimig1 committed:
Corrected metadata

mimig1 committed:
Merge branch 'master' into tocexperiment-toAB

mimig1 committed:
updated to incorporate Connie's feedback

mimig1 committed:
Merge branch 'tocexperiment-toAB' of https://github.com/mimig1/azure-docs-pr into tocexperiment-toAB

Commit:
Adding code for both Node.js 3.0 & 2.2 MongoDB drivers

Commit:
Fixing styling issues

mimig1 committed:
updated experimental value

mimig1 committed:
changed end date of 1st round of experiment

mimig1 committed:
Attempted to correct numbering at end of article

wmgries committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into scripting-quickstarts

Commit:
Tamra feedback

Commit:
Prep for CFR

Commit:
Merge pull request #13 from mimig1/patch-233

Attempted to correct numbering at end of article

wmgries committed:
Convert jq into JMESPATH.

wmgries committed:
Reconcile cynthn/files-ps and wmgries/scripting-quickstarts.

mimig1 committed:
final updates

mimig1 committed:
Merge branch 'master' into tocexperiment-toAB

wmgries committed:
Align to @cynthn's filename.

wmgries committed:
Properly link share snapshots and delete preview tag.

wmgries committed:
Reconcile cynthn/files-quickstart into wmgries/scripting-quickstarts.

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

wmgries committed:
Reconcile cynthn/files-portal into wmgries/scripting-quickstarts.

bwren committed:
Monitoring TOC

wmgries committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into scripting-quickstarts

wmgries committed:
Fix broken links.

bwren committed:
media

bwren committed:
toc

liwong88 committed:
changed TOC

bwren committed:
tab in toc

anoobbacker committed:
Update storsimple-8000-security.md

linda33wj committed:
Refine fault tolerence doc

linda33wj committed:
Update max allowed DMUs and refine parallelCopies instruction

linda33wj committed:
Update lookup activity supported data stores

rboucher committed:
fix TOC issues

bwren committed:
Change to monitoring

bwren committed:
toc

alkohli committed:
Suggested some changes

anoobbacker committed:
Merge pull request #2 from alkohli/patch-185

Suggested some changes

wmgries committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into scripting-quickstarts

wmgries committed:
Merge branch 'files-explorer' into scripting-quickstarts

bwren committed:
toc

bwren committed:
add solution

v-geberr committed:
edits

mimig1 committed:
updates

mimig1 committed:
Merge branch 'tocexperiment-toAB' of https://github.com/mimig1/azure-docs-pr into tocexperiment-toAB

cherylmc committed:
create

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into create

KumudD committed:
updated to cli 2

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ilbcliupdate0326

iainfoulds committed:
Update virtual-machine-scale-sets-upgrade-scale-set.md

v-geberr committed:
edits

tamram committed:
Improving SEO for blobs

KumudD committed:
update image

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ilbcliupdate0326

tamram committed:
fixing links for Disks entry in storage TOC

douglaslMS committed:
Added PK limitation per PM.

KumudD committed:
updated test section

davidmu1 committed:
added local account text

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ilbcliupdate0326

cherylmc committed:
media

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into create

alkohli committed:
Added a section on GDPR compliance

cherylmc committed:
format

bwren committed:
update monitoring overview

KumudD committed:
fixed formatting

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ilbcliupdate0326

v-geberr committed:
new images

wmgries committed:
Clean up of the new storage explorer guide.

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into create

v-geberr committed:
grey background

Commit:
Merge pull request #36412 from iainfoulds/patch-2

[VMSS] VMSS upgrades

Commit:
Merge pull request #36420 from douglaslMS/sync-fix-pk

Added PK limitation per PM.

v-geberr committed:
acro fix

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

mattbriggs committed:
adding new topic for API

GitHubber17 committed:
Merge pull request #36422 from tamram/tamram-0327a

Updating disks TOC, improving SEO for blob introduction article

bwren committed:
fixes

cherylmc committed:
format

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into create

mattbriggs committed:
update titles

anirudhcavale committed:
Updating api-version

nitinme committed:
minor updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-databricks-updates

Juliako committed:
Update media-services-dotnet-get-started-with-aad.md

MGoedtel committed:
updated troubleshooting table per engineering

klaaslanghout committed:
excluded files add to planning doc 

should change to an include for this doc and faq

GitHubber17 committed:
Update azure-stack-rest-api-use.md

Commit:
Merge pull request #36437 from MGoedtel/UpdateVMOMSExt327

updated troubleshooting table per engineering

rwike77 committed:
added video link

GitHubber17 committed:
Merge pull request #36169 from AndrewHoh/patch-32

Adding code for both Node.js 3.0 & 2.2 MongoDB drivers

Commit:
Merge pull request #36438 from klaaslanghout/patch-20

excluded files add to planning doc

wmgries committed:
Share content between Azure Files Storage Explorer and Azure Files Portal.

bandersmsft committed:
cost mgt - adding storage accounts article

wmgries committed:
Clean up TOC.

wmgries committed:
Add redirect for removed file.

wmgries committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into scripting-quickstarts

GitHubber17 committed:
Merge pull request #36430 from mattbriggs/mdb_rest-addition

adding new topic for API

wmgries committed:
Missed file.

bandersmsft committed:
acrolinx fixes

PRMerger12 committed:
Merge pull request #36440 from nitinme/nitinme-databricks-updates

Minor update to Databricks

PRMerger14 committed:
Merge pull request #36441 from rwike77/vidlink

added video link

wmgries committed:
Fix broken links.

rmca14 committed:
Merge pull request #36410 from cherylmc/create

Create

rboucher committed:
Update Alerts section and Action Groups

Small updates.

v-geberr committed:
toc

v-geberr committed:
edits

PRMerger20 committed:
Merge pull request #36444 from rboucher/patch-17

Update Alerts section and Action Groups

rmca14 committed:
Merge pull request #36442 from bandersmsft/master

cost mgt - adding storage accounts article

rboucher committed:
another small update.

wmgries committed:
Reorder/clean up storage explorer page.

wmgries committed:
Fix broken checklists.

wmgries committed:
Fix CLI formatting.

shawntabrizi committed:
Updating MS Date and adding new item to TOC

BryanTrach-MSFT committed:
MicrosoftDocs/azure-docs#6382

Updated wording under sub-section "External admin takeover" due to extra words.

j-martens committed:
AML: redirecting renamed pricing file

j-martens committed:
AML: Adding a regional page

Commit:
Merge pull request #36432 from anirudhcavale/patch-5

Updating api-version

christiankuhtz committed:
tweak

j-martens committed:
AML: renaming pricing page

PRMerger10 committed:
Merge pull request #36448 from BryanTrach-MSFT/patch-9

MicrosoftDocs/azure-docs#6382

PRMerger15 committed:
Merge pull request #36451 from christiankuhtz/patch-112

tweak

liwong88 committed:
added breadcrumbs

Commit:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr

Commit:
wave 4

rjagiewich committed:
Merge pull request #36406 from v-geberr/0327-qs-hier-entity

quickstart hier-entity

Commit:
header updates

Commit:
Merge pull request #36433 from Juliako/patch-6

Update media-services-dotnet-get-started-with-aad.md

j-martens committed:
Removing regions

rjagiewich committed:
Merge pull request #36449 from shawntabrizi/master

Updating MS Date and adding new item to TOC

rjagiewich committed:
Merge pull request #33879 from hning86/master

spaces

rjagiewich committed:
Merge pull request #36333 from lisawong19/cog-content-moderator

changed TOC to MVC style

KumudD committed:
feedback incorp

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ilbcliupdate0326

KumudD committed:
fixed Acrolinx issues

rboucher committed:
fix TOC error. Move stats article to reference

rboucher committed:
wording change

KumudD committed:
updated TM limits

rboucher committed:
wording change

rboucher committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TOCfix

rboucher committed:
fix TOC again

ajaycode committed:
added Traffic Analytics

KumudD committed:
removed preview text

rboucher committed:
fix broken link

ajaycode committed:
added Traffic Analytics

rboucher committed:
fix typo. Darn it.

PRMerger17 committed:
Merge pull request #36459 from KumudD/removepreviewtext

Remove preview text

zhiweiwangmsft committed:
Update adfs audit logs faq

zhiweiwangmsft committed:
Update audit logs lines

anoobbacker committed:
Update storsimple-8000-security.md

PRMerger13 committed:
Merge pull request #36458 from ajaycode/network-monitoring-overview

added Traffic Analytics

PRMerger15 committed:
Merge pull request #36462 from zhiweiwangmsft/patch-26

Update adfs audit logs faq

rboucher committed:
corrected a few typos and issues

Commit:
Merge pull request #36447 from rboucher/patch-19

another small update.

anoobbacker committed:
Update storsimple-8000-security.md

Format changes

anoobbacker committed:
Update storsimple-8000-security.md

Text corrections.

bwren committed:
remove en-in

Commit:
Updates.

Commit:
Fixes.

linda33wj committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

linda33wj committed:
Add tips for HTTP/Web connector, and add type mapping for ORC/Parquet format

JeffGoldner committed:
Fix bad link in PTAF

PRMerger15 committed:
Merge pull request #36342 from anoobbacker/patch-30

Update storsimple-8000-security.md

mimig1 committed:
Addressing blocking issues.

mmacy committed:
[AKS] fix cli link

Commit:
Merge pull request #36424 from davidmu1/gitissue5868

added local account text

Commit:
Merge pull request #36471 from JeffGoldner/master

Fix bad link in PTAF

Commit:
Merge pull request #36472 from mmacy/aks-link-fix

[AKS] fix cli link

rjagiewich committed:
Merge pull request #36423 from KumudD/ilbcliupdate0326

Internal Load Balancer - Azure CLI 2.0

rjagiewich committed:
Merge pull request #36317 from bwren/monitoring-toc

Monitoring TOC

PRMerger12 committed:
Merge pull request #34481 from dhaval24/Fix-ApplicationInsights-Java-GettingStarted

Fix Documentation for Application Insights Java SDK

mimig1 committed:
Fixed almost all non blocking issues and incorped PR #36436

rjagiewich committed:
Merge pull request #36450 from j-martens/master

 AML: Adding a regional page

rjagiewich committed:
Merge pull request #36470 from jimdial/patch-119

Updates.

davidmu1 committed:
added toc nodes

linda33wj committed:
Fix typo

mimig1 committed:
Cosmos DB: Added links from QS to SQL intro

KumudD committed:
added metadata

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into TMLimitsUpdate0327

wmgries committed:
Add metadata for includes files per @Ja-Dunn.

alkohli committed:
Minor format change

Commit:
whatsnew07

PRMerger10 committed:
Merge pull request #36475 from mimig1/03282017

Cosmos DB: Added links from QS to SQL intro

itechedit committed:
Merge pull request #36347 from linda33wj/master

Update ADF content

linda33wj committed:
Tweak wording based on feedback

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

mimig1 committed:
Cosmos DB: Added links from QS to Intro

rjagiewich committed:
Merge pull request #36456 from KumudD/TMLimitsUpdate0327

update TM limits

mimig1 committed:
Cosmos DB: 2 small H1 consistency changes

itechedit committed:
Merge pull request #33591 from wmgries/scripting-quickstarts

Azure Files scripting quickstarts

linda33wj committed:
Tweak wording

PRMerger17 committed:
Merge pull request #36479 from mimig1/03282018mongo

Cosmos DB: Added links from QS to Intro

v-alje committed:
Merge pull request #36480 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger17 committed:
Merge pull request #36425 from alkohli/patch-186

Added a section on GDPR compliance

PRMerger18 committed:
Merge pull request #36477 from MarkusVi/whatsnew07

whatsnew07

rjagiewich committed:
Merge pull request #36354 from rboucher/TOCfix

fix TOC issues

PRMerger19 committed:
Merge pull request #36481 from mimig1/03282018table

Cosmos DB: 2 small H1 consistency changes

itechedit committed:
Merge pull request #36476 from davidmu1/gitissue5546

added toc nodes

GitHubber17 committed:
Merge pull request #35949 from mimig1/tocexperiment-toAB

Cosmos DB: TOC & index for public A/B experiment

Commit:
Merge pull request #36478 from linda33wj/master

Tweak wording based on feedback

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
